sass/_ExplanationModal.scss

```diff
@@ -3,6 +3,10 @@
 .coveo-user-explanation-modal {
   $modal: &;
 
+  .coveo-small-close {
+    display: none;
```

Comment:
Since there's already a cancel button, I think this is fine. We should probably make sure that an element inside the modal gets focused when the modal is opened though.